src/Browser.php

```diff
@@ -725,6 +728,49 @@ public function withResponseBuffer($maximumSize)
         ));
     }
 
+    /**
+     * The `withHeader(string $header, string $value): Browser` method can be used to
+     * add a request header for all following requests.
```

Comment:
We usually use the first sentence to describe the full name of the function only in our `README.md` and write a shorter version in here,  because you can still read the full function name a few lines below (also stays consistent with the other docblocks). Maybe change it to this:
```suggestion
     * Add a request header for all following requests.
```

Same goes for the `withoutHeader()` method.